docs/fsharp/language-reference/generics/statically-resolved-type-parameters.md

```diff
@@ -61,6 +60,32 @@ The output is as follows.
 1.500000
 ```
 
+Starting with F# 4.1, you can also specify concrete type names in statically resolved type parameter signatures.  In previous versions of the language, the type name could actually be inferred by the compiler, but could not actually be specified in the signature.  This quirk in the language is addressed in F# 4.1, but is also backwards-compatible.  Here's an example:
```

Comment:
I think it's best to generally avoid references to version numbers in the docs, but I'm not sure.  They tend to look dated pretty quickly, e.g. references to "F# 3.0.1" would look pretty odd in the docs today.  But it's ok for now, not a big deal.

Also generally avoid words like "quirk" and "backwards-compatible". So something like this is sufficient:

> As of F# 4.1, you may also specify concrete type names in statically resolved type parameter signatures.  Here's an example: